connexion/apis/abstract.py

```diff
@@ -128,12 +128,17 @@ def __init__(self, specification, base_path=None, arguments=None,
         # API calls.
         self.consumes = self.specification.get('consumes', ['application/json'])  # type: List[str]
 
-        self.security = self.specification.get('security')
-        self.security_definitions = self.specification.get('securityDefinitions', dict())
-        logger.debug('Security Definitions: %s', self.security_definitions)
-
         self.definitions = self.specification.get('definitions', {})
         self.components = self.specification.get('components', {})
+
+        self.security = self.specification.get('security', dict())
```

Comment:
Default should be `list()`. But the change isn't critical as there is a 'boolean' check before the list gets accessed.